payments/infrastructure/order_repositroy.py

```diff
@@ -36,8 +36,6 @@ def add_item(self, item):
             self.session.rollback()
             raise e
 
-        self.session.commit()
```

Comment:
@pratik-vii @prashantramangupta why this change?

Reply:
removed double commits